### ISSUE TITLE
`leftContent` input no longer required

### ISSUE THREAD

**Issue by danielsharvey** [ng-doc/ng-doc#203]
The `leftContext` is mentioned in the docs and included in the generators, but looks to be no longer required.

https://github.com/ng-doc/ng-doc/blob/1127d1c5ffe9cc7a078b9bfacc685a9f9b187650/apps/ng-doc/docs/getting-started/configuration/index.md?plain=1#L188-L192

**Comment by skoropadas** [comment]
Good catch thanks, I'll update it, it's not an input it's ng-content selector now.
